src/cms-content/about/index.tsx

```diff
@@ -7,29 +7,43 @@ export interface LogoItem {
   url: string;
 }
 
-interface PartnersContent {
+export interface CommitmentsContent {
+  icon: string;
+  altText: string;
+  copy: Markdown;
+}
+
+export interface PartnersContent {
+  header: string;
   copy: Markdown;
   logos: LogoItem[];
 }
 
 interface AboutContent {
-  pageHeader: string;
-  introHeader: string;
-  introContent: Markdown;
+  pageHeader: string; // deprecated
```

Comment:
given the amount of newly deprecated fields and content, might be good to remove them from the interfaces//config//cms altogether. or, if you want to punt to a later pr, thats cool too

Reply:
I'm not super confident in sequencing things between code/CMS so I think I prefer to save that for a follow-up

Reply:
sounds good